download.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Download
+summary: Download the most officially maintained versions of TiDB enterprise tools.
+category: resources
+---
+
+# Download
+
+This document collects the available downloads for most officially maintained versions of TiDB enterprise tools.
+
+## TiDB Enterprise Tools
+
+| Package name | OS | Architecture | SHA256 checksum |
+|:---|:---|:---|:---|
+| [tidb-enterprise-tools-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-enterprise-tools-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.sha256) |
+
+This enterprise tools package includes all the following tools:
+
+- Syncer
+- Loader
+- mydumper
+- binlogctl
+- checker
+- ddl_checker
+- dump_region
+- importer
+- sync_diff_inspector
+
+## TiDB-Binlog
```

Comment:
remove ` TiDB-Binlog` and `TiDB-Lightning` tarball downlink?  they are also in `tidb` tarball, let user to get it from `tidb` tarball

Reply:
But the user might do not know about this. A separate link can be more friendly.

Reply:
we always want to provide one tarball that contains all binary user need